AdminGUI/package.json

```diff
@@ -2,39 +2,52 @@
   "name": "adminGui",
   "version": "1.0.0",
   "scripts": {
+    "tsc": "tsc"
   },
   "dependencies": {
-    "@angular/common": "2.0.0",
-    "@angular/compiler": "2.0.0",
-    "@angular/core": "2.0.0",
-    "@angular/forms": "2.0.0",
-    "@angular/http": "2.0.0",
-    "@angular/platform-browser": "2.0.0",
-    "@angular/platform-browser-dynamic": "2.0.0",
-    "@angular/router": "3.0.0",
-    "@angular/upgrade": "2.0.0",
-
-
-    "systemjs": "0.19.27",
-    "es6-shim": "^0.35.0",
-    "reflect-metadata": "^0.1.3",
-    "rxjs": "5.0.0-beta.6",
-    "zone.js": "^0.6.12",
-
-    "angular2-in-memory-web-api": "0.0.7",
+    "@angular/common": "^2.2.0",
+    "@angular/compiler": "^2.2.0",
+    "@angular/core": "^2.2.0",
+    "@angular/forms": "^2.2.0",
+    "@angular/http": "^2.2.0",
+    "@angular/platform-browser": "^2.2.0",
+    "@angular/platform-browser-dynamic": "^2.2.0",
+    "@angular/router": "^3.2.0",
+    "@angular/upgrade": "^2.2.0",
+    "@types/node": "^10.9.4",
     "angular2-cookie": "^1.2.0",
-    "bootstrap": "^3.3.6",
+    "angular2-in-memory-web-api": "0.0.7",
+    "basecode": "1.2.0",
+    "bootstrap": "3.3.6",
+    "es6-shim": "0.35.0",
     "moment": "2.13.0",
-    "ng2-file-upload": "^1.0.3",
+    "ng2-file-upload": "1.0.3",
     "primeng": "1.0.0-beta.15",
     "primeui": "4.1.12",
-	"typescript" : "2.0.3"
+    "reflect-metadata": "0.1.3",
+    "rxjs": "5.0.0-beta.12",
+    "systemjs": "0.19.39",
+    "typescript": "2.0.3",
+    "zone.js": "0.6.26"
   },
   "devDependencies": {
-    "gulp": "3.8.11",
+    "babel-core": "",
+    "babel-plugin-transform-cjs-system-wrapper": "",
+    "bluebird": "",
+    "data-uri-to-buffer": "",
+    "es6-template-strings": "",
+    "glob": "",
+    "gulp": "3.9.1",
     "gulp-typescript": "",
-    "rimraf": "2.2.8"
+    "gulp-util": "",
+    "mkdirp": "",
+    "pure-cjs": "",
+    "rimraf": "2.2.8",
+    "source-map": "",
+    "systemjs-builder": "",
+    "traceur": "",
+    "typescript": "^2.0.3",
```

Comment:
typescript version is not consistent with the one specified in `dependencies` section. Will that be an issue?

Reply:
It should be fixed to 2.0.3.